src/app/store/index.ts

```diff
@@ -1,11 +1,28 @@
-import { FirstState } from './first';
-import { SecondState } from './second';
+import { createStore, applyMiddleware, compose } from 'redux';
+import createSagaMiddleware from 'redux-saga';
+// import { createEpicMiddleware } from 'redux-observable';
 
-// only reducers from ./store
-export interface RootStore {
-  first: FirstState;
-  second: SecondState;
-}
+import rootReducer from './rootReducer';
+// import rootEpic from './rootEpic';
+import rootSaga from './rootSaga';
 
-export * from './first';
-export * from './second';
+// const epicMiddleware = createEpicMiddleware();
+const sagaMiddleware = createSagaMiddleware();
+
+// @ts-ignore
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
I believe that with `(window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__` you can get rid of ts-ignore comment